.github/workflows/python-test.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         - name: Check formatting with black
           run: black . --check --verbose
         - name: Run Python tests
-          run: python3 -m pytest
+          run: python3 -m pytest --cov
```

Comment:
is this the coverage flag? I see you removed it from an config file below

Reply:
Yes, good catch. I removed it from the overall configuration as it seems to be incompatible with running a single module (as I'm trying to do with the e2e tests). This means that you have to specify it when you want --cov so I put it in the README. I can explore other workarounds for running just the E2E tests if you're worried about adding --cov manually. Maybe keyword would work.